app/views/internal/shared/_navbar.html.erb

```diff
@@ -1,9 +1,34 @@
-<% menu_items = %w[comments articles users tags welcome broadcasts reports response_templates pages tools chat_channels permissions growth mods config events badges organizations sponsorships podcasts negative_reactions moderator_actions].sort.each_with_object({}) { |v, h| h[v] = v } %>
-<% menu_items[:listings] = "listings" %>
-<% menu_items[:webhooks] = "webhook_endpoints" %>
+<% menu_items = %w[
+     articles
+     badges
+     broadcasts
+     chat_channels
+     comments
+     config
+     events
+     growth
+     listings
+     moderator_actions
+     mods
+     negative_reactions
+     organizations
+     page_redirects
+     pages
+     permissions
+     podcasts
+     reports
+     response_templates
+     sponsorships
+     tags
+     tools
+     users
+     welcome
+   ].each_with_object({}) { |v, h| h[v] = v } %>
+
+<% menu_items["webhooks"] = "webhook_endpoints" %>
 
 <div class="mt-4">
-  <% menu_items.each do |name, controller| %>
+  <% menu_items.sort_by { |name, _controller| name }.each do |name, controller| %>
```

Comment:
Added this to alphabetize the menu times so they display alphabetically - more user friendly.